.github/workflows/ci.yml

```diff
@@ -12,32 +12,16 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Checkout tools repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          repository: emberjs/rfcs-tooling
+          repository: mansona/emberjs-rfcs-tooling
+          ref: new-format-frontmatter
```

Comment:
I can drop this change once https://github.com/emberjs/rfcs-tooling/pull/4 is merged 👍

Reply:
this has been dropped 👍